index.html

```diff
@@ -2,9 +2,8 @@
 layout: null
```

Comment:
You need to add `title: Verovio` for seo

Reply:
You mean in the `_config.yml`?

Reply:
Nevermind.